sample-plugins/sample-namespace/plugin-gotd/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "private": true,
+  "name": "plugin-gotd",
+  "version": "1.44.0",
+  "main": "extension.js",
```

Comment:
This `main` field is extraneous, right? Relevant in a traditional plugin (VS Code externsion), but not in this case.

Reply:
Yeah, I forgot to delete that when I reworked the plugin as a dual headless/backend plugin. It would also work for a headless plugin using the `theiaHeadless` engine but that's not the case, here.

Reply:
Ah, the `"main"` entrpoint is back now that this example plugin is a VS Code plugin. 😉